### ISSUE TITLE
HttpClient is deprecated on Android 6.0

### ISSUE THREAD

**Issue by raghunandankavi2010** [Microsoft/Cognitive-Face-Android#50]
The sample app uses httpclient which is deprecated in android 6.0. I think you should switch to HttpUrlConnection instead.

**Comment by scognito** [comment]
As workaround you can edit the Android Manifest this way.
Inside the <application> tag add this tag:
```
<uses-library
            android:name="org.apache.http.legacy"
            android:required="false" />
```
See [this](https://developer.android.com/about/versions/pie/android-9.0-changes-28#apache-p) for detail.

**Comment by raghunandankavi2010** [comment]
i have done that already. i opened the issue cause i wanted the authors to port to httpurlconnection.

**Comment by lebronJ** [comment]
We have fixed this with PR #53 merged, which replaces outdated `httpClient` with `OkHttp`. Feel free to reopen this if you still meet similar problems.
